router/h2/src/e2e/scala/io/buoyant/router/h2/RouterEndToEndTest.scala

```diff
@@ -147,11 +149,11 @@ class RouterEndToEndTest
       assert(!rspF.isDefined)
 
       dogRspP.setValue(serverLocalStream)
+      serverLocalQ.offer(Frame.Data("boop", eos = false))
       val rsp = await(rspF)
       val reqReadF = reqStream.read()
       val rspReadF = rsp.stream.read()
 
-      rspReadF.raise(new ChannelClosedException())
```

Comment:
This seemingly fixed a flaky test failure:
```
Throw(Reset.Closed) did not equal Throw(Reset.Cancel) (RouterEndToEndTest.scala:159)
```
But I don't really understand why this was here in the first place or how it could have non-deterministically caused that failure.